experiments/templates/experiments/mail/stats.txt

```diff
@@ -0,0 +1,4 @@
+Dear lab-staff,
+
+Beep boop,
+The ILS web-experiment datastore
```

Comment:
At this point, why even include a file :P The auto-generated one will be better

It's funny tho